### PR TITLE
Added daisy

### PR DESCRIPTION
**Pull request by anandhupm**

### CODE REVIEW

File: tutorial/tutorial/Controllers/WeatherForecastController.cs [hunk 1]
```diff
@@ -8,7 +8,7 @@ public class WeatherForecastController : ControllerBase
     {
         private static readonly string[] Summaries = new[]
         {
-        "Freezing", "Bracing", "Chilly", "Cool", "Mild", "Warm", "Balmy", "Hot", "Sweltering", "Scorching"
+        "Freezing", "Bracing", "Chilly", "Cool", "Mild", "daisy", "Balmy", "Hot", "Sweltering", "Scorching"
```
Comment (anandhupm) [inline tutorial/tutorial/Controllers/WeatherForecastController.cs:11]: why this change?

Comment (anandhupm) [reply tutorial/tutorial/Controllers/WeatherForecastController.cs:11]: Unnecessary?

**Review by shruthty38939393 — APPROVED** [review]
